torchao/float8/README.md

```diff
@@ -269,3 +269,37 @@ python test/float8/test_fsdp2/test_fsdp2.py
 # make sure to turn on torch.compile to get the best performance
 ./benchmarks/float8/bench_linear_float8.py -o ../tmp/test.txt --compile
 ```
+
+### Training benchmarks
+
+[Torchtitan](https://github.com/pytorch/torchtitan) was used to benchmark float8 training performance, for both rowwise
+and tensorwise scaling. The training benchmarks were all run using:
+
+- Single-node training on 8xH100 GPUs
+- Batch size 2
+- Sequence length 8192
+- Steps 100
+- `torch.compile`
+- FSDP2
+
+| Model         | Scaling     | Activation checkpointing | Average tokens/second     | Peak Memory (GB) |
+| ------------- | ----------- | ------------------------ | ------------------------- | ---------------- |
+| Llama3-8b     |  tensorwise | per op SAC               | 3217.4                    | 75.47            |
```

Comment:
The TPS number looks a bit low, I was expecting something in the 7k range.  What machine are you using?  I'm currently on `devgpu006.vll6` and I've noticed in the past couple of weeks torchtitan performance has been about 0.5x of what it was ~weeks ago, so it's either something local to my machine or a regression somewhere in the stack.

Reply:
> What machine are you using? I'm currently on `devgpu006.vll6`

I'm on this machine as well, I'll try using a new one.

> hmm, or this may be related to batch_size 2 I see on L279. Was this changed from torchtitan defaults?

default batch size is 1, I actually increased it to 2 to get the TPS as high as we could within the memory capacity constraint of an H100. I also was thinking batch size of 1 is probably not super common for production training runs so was trying to make it more representative of real workloads.

Reply:
> I'm currently on devgpu006.vll6 and I've noticed in the past couple of weeks torchtitan performance has been about 0.5x of what it was ~weeks ago

Good catch, I switched machines and perf was much better. Updated PR accordingly